app/players/elves/druid.py

```diff
@@ -0,0 +1,19 @@
+from .elf import Elf
+
+
+class Druid(Elf):
+
+    def __init__(
+        self, nickname: str, musical_instrument: str, favourite_spell: str
+    ) -> None:
+        super().__init__(nickname, musical_instrument)
+        self._favourite_spell = favourite_spell
+
+    def player_info(self) -> str:
+        return (
+            f"Druid {self.nickname}. "
+            f"{self.nickname} has a favourite spell: {self._favourite_spell}"
+        )
+
+    def get_rating(self) -> None:
+        return len(self._favourite_spell)
```

Comment:
The `get_rating` method is annotated to return `None`, but it actually returns an `int`. You should change the return type annotation to `int` to match the actual return type.